src/sparql/reports/gard-mondo-mapped-obsoletes.sparql

```diff
@@ -0,0 +1,17 @@
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+SELECT ?cls ?cls_label ?xref ?comment WHERE {
+  ?cls a owl:Class ;
+    rdfs:label ?cls_label ;
+    oio:hasDbXref ?xref ;
+    rdfs:comment ?comment ;
+    owl:deprecated "true"^^xsd:boolean .
+
```

Comment:
```suggestion
OPTIONAL {
?cls <http://purl.obolibrary.org/obo/IAO_0000231> ?obsolesence_reason .
}
```

IAO:0000231 is the true obsolescence reason, but you are right, comment is most of the time more informative. (and add to table columns).

Reply:
Ah, I overlooked that. That is also now included!

All done now and ready for you to look at again.